Multiplayer/Components/Networking/PlayerlistGUI.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Cysharp.Threading.Tasks;
+using JetBrains.Annotations;
+using Multiplayer.Networking.Listeners;
+using UnityEngine;
+using UnityModManagerNet;
+
+namespace Multiplayer.Components.Networking;
+
+public class PlayerlistGUI : MonoBehaviour
+{
+    [CanBeNull]
+    private NetworkClient _client;
+    private bool showPlayerlist;
+
+    public void Show([CanBeNull] NetworkClient client)
```

Comment:
Instead of passing in a NetworkClient, you can use `NetworkLifecycle.IsClientRunning` and `NetworkLifecycle.Client`.

Reply:
Yes this works, I changed it in my newest Commit